File: Selenium/Activities/Activity4.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
    
    driver.get("https://v1.training-support.net/selenium/target-practice")

    print(driver.title)

    print(driver.find_element(By.XPATH, "//h3[@id='third-header']").text)

    print(driver.find_element(By.XPATH, "//h5[contains(text(),'Fifth header')]").value_of_css_property("color"))

    print(driver.find_element(By.XPATH, "//button[contains(text(),'Violet')]").get_attribute("class"))

    print(driver.find_element(By.XPATH, "//button[@class='ui grey button']").text)

    driver.close()

File: Selenium/Activities/Activity4.java
package activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity4 {

	public static void main(String[] args) {
		
		WebDriver driver =  new FirefoxDriver();
		
		driver.get("https://v1.training-support.net/selenium/target-practice");
		
		System.out.println(driver.getTitle());
		
		System.out.println(driver.findElement(By.xpath("//h3[@id='third-header']")).getText());
		
		System.out.println(driver.findElement(By.xpath("//h5[contains(text(),'Fifth header')]")).getCssValue("color"));
		
		System.out.println(driver.findElement(By.xpath("//button[contains(text(),'Violet')]")).getAttribute("class"));
		
		System.out.println(driver.findElement(By.xpath("//button[@class='ui grey button']")).getText());
		
		driver.close();

	}

}
